cpu/cc2538/vectors.c

```diff
@@ -71,74 +71,74 @@ WEAK_DEFAULT void isr_dmaerr(void);
 /* interrupt vector table */
 ISR_VECTORS const void *interrupt_vector[] = {
     /* Exception stack pointer */
-    (void*) (&_estack),             /* pointer to the top of the stack */
+    __extension__ (void*) (&_estack),             /* pointer to the top of the stack */
```

Comment:
Isn't the root cause for the warning that ISO C forbids putting a function pointer in void*? Silencing the warning like this seems closing eyes, and will cause problems whenever -pedantic might prove useful, e.g., when actually relying on the ISO C compliance.